doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,167 @@
+# Solve a system of inequalities of a single variable algebraically
+
+Use SymPy to solve a system of univariate inequalities algebraically. For
+example, solving $x^2 < \pi$, $x > 0$ yields $0 < x < \sqrt{\pi}$.
+
+```{note}
+SymPy can currently solve inequalities involving only one variable (symbol).
+```
+
+Alternatives to consider:
+- For multivariate systems (more than one symbol), try this [SciPy linear
+  programming
+  function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.linprog.html)
+
+Here is a simple example of solving a system of inequalities of a single
+variable algebraically. solve {func}`~.solve` accepts a list or tuple of
+inequalities to be solved as a system:
+
+```py
+>>> from sympy import symbols, solve, pi
+>>> x = symbols('x')
+>>> solve([x >= 0, x**2 <= pi], x)
+(0 <= x) & (x <= sqrt(pi))
+```
+
+## Guidance
+
+### Include the variable to be solved for in the function call
+
+We recommend you include the variable to be solved for as the second argument
+for {func}`~.solve`. While {func}`~.solve` can currently solve systems of
+equations with only one symbol, it is a good practice in case that capability is
+expanded, and because {func}`~.solve` can solve an equation with more than one
+symbol.
+
+## Solve a system of inequalities algebraically
+
+You can create your inequalities, then solve the system as a list:
+
+```py
+>>> from sympy import symbols, solve, pi, Integer
+>>> x = symbols('x')
+>>> one_third = Integer(1)/Integer(3)
+>>> solve([x >= one_third, x**2 <= pi], x)
+(1/3 <= x) & (x <= sqrt(pi))
+```
+
+## Use the solution result
+
+A common way to use the solution result is to extract the bounds for the symbol
+(variable). For example, for a solution of $0 < x < \sqrt{\pi}$, you might want
+to extract $0$ and $\sqrt{\pi}$.
+
+### Extract a list of decomposed relations
+
+You can decompose a set of relations which is joined by `^` (or) or `&` (and)
+into individual relations using relational atoms. Using {any}`canonical
+<sympy.core.relational.Relational.canonical>` will put order each relation so
+the symbol is on the left, so you can take the right-hand side {any}`rhs
+<sympy.core.relational.Relational.lhs>` to extract the constants:
+
+```py
+>>> from sympy import symbols, solve, Integer, pi
+>>> from sympy.core.relational import Relational
+>>> x = symbols('x')
+>>> one_third = Integer(1)/Integer(3)
```

Comment:
same here; otherwise consider using the more ubiquitous `S(1)/3` or `Rational(1, 3)`.